lsp/nls/src/requests/hover.rs

```diff
@@ -57,10 +57,13 @@ pub fn handle(
                     language: "nickel".into(),
                     value: ty.to_string(),
                 }),
-                MarkedString::LanguageString(LanguageString {
-                    language: "plain".into(),
-                    value: meta.join("\n"),
-                }),
+                MarkedString::String(
+                    meta.iter()
+                        .flat_map(|s| s.lines())
+                        .map(|s| s.trim())
```

Comment:
This seems to trim the identation from code snippets, which is probably not intended.

Reply:
True, it seems like the desired behavior should be like that of the `indoc!` macro which removes only the common indentation in the lines. Should I add the implementation for that macro, [unindent](https://github.com/dtolnay/indoc/tree/master/unindent), as a dependency, or make a seperate function for that?

Reply:
Since `unindent` is in our dependency tree already for `indoc`, using that seems like a reasonable thing to do 👍